### PR TITLE
Implement Crc64 as a struct with Write

### PR DESCRIPTION
**Pull request by seppo0010**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -144,7 +146,46 @@ pub fn crc64(crc: u64, data: &[u8]) -> u64 {
...
+pub struct Crc64 {
+    crc64: u64,
+}
+
+impl Crc64 {
+    pub fn new() -> Crc64 {
+        Crc64 { crc64: 0 }
+    }
+    pub fn get(&self) -> u64 {
+        self.crc64.clone()
```
Comment (badboy) [inline src/lib.rs]: Why clone? It should do that anyway, right?

Comment (seppo0010) [inline src/lib.rs]: You are right, it is not necessary

**Comment by badboy** [comment]
@homu retry

**Comment by badboy** [comment]
@homu r+

**Pull request merged**